components/raftstore/src/store/worker/read.rs

```diff
@@ -418,63 +524,71 @@ impl Progress {
     }
 }
 
-pub struct LocalReader<C, E>
+pub struct LocalReader<C, E, D, S>
```

Comment:
please add a comment to indicate that LocalReader is the entry class to handle read all tablets's data. (Basically it's per store's reader)

Reply:
Fine.